app/models/community_resource.rb

```diff
@@ -31,20 +33,48 @@ def self.published
     )
   end
 
+  def title; description; end
+  def self.matchable; published; end
+
   def published?
     now = Time.current
     is_approved &&
       (publish_from.present? ? publish_from <= now : true) &&
       (publish_until.nil? || now < publish_until)
   end
 
+  def categories_for_tags
+    Category.where(name: tag_list)
+  end
+
   def all_tags_unique
     all_tags_list.flatten.map(&:downcase).uniq
   end
 
   def all_tags_to_s
     all_tags_unique.join(', ')
   end
+
+  def preferred_contact_method
+    # TODO: This is a hack that makes things work for now
+    # The test creates a contact method that will match this
+    # and the dev db seeding creates a couple, too
+    ContactMethod.method_name('call').last
+  end
+
+  def type
+    "Community Resource"
+  end
+
+  def inexhaustible
+    true
+  end
+
+  def urgency_level_id
+    UrgencyLevel::TYPES.last.id
+  end
```

Comment:
@exbinary mentioned that these don't feel like actual attributes, behaviors, or properties of a community resource, and so they might be better off in an abstraction or interface class that turns community resources into something blueprint-able